docs/providers/documentation/gke-provider.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Google Kubernetes Engine (GKE)"
```

Comment:
Looks like title is making tests to fail: https://github.com/keephq/keep/actions/runs/11316619299/job/31469161221?pr=2167
```suggestion
title: "GKE" 
```

Reply:
updated